hack/deploy/catalogsource.yaml

```
@@ -0,0 +1,11 @@
+---
+kind: CatalogSource
+apiVersion: operators.coreos.com/v1alpha1
+metadata:
+  name: mcg-osd-deployer-catalog
+  namespace: redhat-data-federation
+spec:
+  image: quay.io/rexagod/mcg-osd-deployer:catalog-00
```

```suggestion
  image: quay.io/osd-addons/mcg-osd:catalog-console-ci
```
Will make this change.